packages/vue/src/utils/has-slot.ts

```diff
@@ -8,14 +8,16 @@ import { renderSlotText } from "./render-slot-text";
  * @param vm - Component instance.
  * @param name - Name of the slot to check for.
  * @param props - Props required by a scoped slot.
+ * @param options - Render options.
  * @returns true if the slot is implemented and have non-empty content.
  */
 export function hasSlot(
     vm: { $slots: Slots },
     name: string,
     props: Record<string, unknown> = {},
+    options: Partial<RenderSlotOptions> = {},
```

Comment:
@KrQnan66140124 Nu har jag precis gjort ändingar i `hasSlot`, är det något som påverkar dina ändringar?

Reply:
@ext Tror det som kan hända är att jag behöver justera metoden i composablen men det ska gå lätt om den är in mergad